backend_test.go

```diff
@@ -366,6 +376,12 @@ func testDBBatch(t *testing.T, backend BackendType) {
 	require.NoError(t, batch.Set([]byte("b"), []byte{2}))
 	require.NoError(t, batch.Set([]byte("c"), []byte{3}))
 	require.NoError(t, batch.Delete([]byte("c")))
+
+	batchSize, err = batch.GetByteSize()
+	require.NoError(t, err)
+	// batchSize should be grater than 10 which is the total size of keys and values
+	require.GreaterOrEqual(t, batchSize, uint64(10))
```

Comment:
Wdyt about adding some kind of randomized test that inserts keys and values of different sizes to batch?

While inserting, we keep a counter of bytes in the test and then compare it to the result of  `GetrBatchSize()`

Reply:
I think that makes sense, I'll add that